tests/test_study.py

```diff
@@ -408,6 +408,13 @@ def test_study_pickle():
     assert len(study_2.trials) == 20
 
 
+def test_study_trials_dataframe_with_no_trials():
+    # type: () -> None
+    study_with_no_trials = optuna.create_study()
```

Comment:
https://github.com/pfnet/optuna/pull/574/files/2c580b44067723046ef45466d31362bba951fe5d#r330981416

```suggestion

    study_with_no_trials = optuna.create_study()
```